frontend/src/components/DataDashboard.vue

```diff
@@ -29,52 +35,42 @@ const downloadCSV = (gender) => {
     const url = window.URL.createObjectURL(blob);
     const a = document.createElement('a');
     a.setAttribute('href', url);
-    a.setAttribute('download', genderToDcid(gender) + '.csv');
+    a.setAttribute('download', `${dcid}.csv`);
     a.click();
   } catch (err) {
     console.log(err);
   }
   loading_download.value = false;
 }
-
-let apiCache = new ApiCache();
-
-async function getData(dcid: string) {
-
-  // This uses DataCommons' public API key. DO NOT INCLUDE A PRIVATE API KEY HERE!
-  let request = "https://api.datacommons.org/v1/observations/series/country/USA/" + dcid + "?key=AIzaSyCTI4Xz-UW_G2Q2RfknhcfdAnTHq5X5XuI";
-
-  if (apiCache.recordExists(request)) {
-    tableItems.value = apiCache.get(request).observations;
+ async function getData(dcid: string) {
+     const values = await dcClient.getData(dcid)
+     tableItems.value = values
   }
 
-  else {
-    const res = await fetch(request);
-    const finalRes = await res.json();
-    tableItems.value = finalRes.observations;
-    apiCache.set(request, finalRes);
-  }
-}
-
-function genderToDcid(gender: string): string {
-  let dcidMap = new Map<string, string>([
-    ["Males", "Count_Person_25OrMoreYears_EducationalAttainmentBachelorsDegree_Male"],
-    ["Females", "Count_Person_25OrMoreYears_EducationalAttainmentBachelorsDegree_Female"]
-  ]);
-
-  let dcid = dcidMap.get(gender);
-  return dcid == undefined ? "" : dcid;
-}
-
-watchEffect(async () => {
-  getData(genderToDcid(gender.value))
-}) 
+ const genderToDcid = (gender: string): string => {
+     let query: string = 'Female'
```

Comment:
IIUC this is the default value we'd load and cache from DC in the open explore case where no initial checkbox is selected. Is that correct?

Reply:
gender is assigned to 'Males' by default on L13.

Reply:
I've added query dimension constants.